examples/compiler/00_index.ipynb

```diff
@@ -0,0 +1,53 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Devito's Operator \n",
```

Comment:
Drop 's I think. Or maybe better "The Devito compiler"

Reply:
fixed